__tests__/ViewModel-tests/map/mapScreenViewModel.test.tsx

```diff
@@ -97,12 +81,17 @@ describe("useMapScreenViewModel", () => {
     });
 
     const { result } = renderHook(() =>
-      useMapScreenViewModel(mockFirestoreCtrl, mockNavigation),
+      useMapScreenViewModel(mockFirestoreCtrl, mockNavigation, undefined),
```

Comment:
This is not very clear, could your precise it's for the groups that are not used in the tests for now ?